b2share/modules/records/b2share_epic.py

```diff
@@ -0,0 +1,90 @@
+import httplib2
+from simplejson import dumps as jsondumps
+from werkzeug.exceptions import abort
+from flask import current_app
+from urllib.parse import urljoin, urlparse
+
+from .errors import EpicPIDError
+
+
+def createHandle(location, checksum=None, suffix=''):
+    """ Create a new handle for a file.
+
+    Parameters:
+     location: The location (URL) of the file.
+     checksum: Optional parameter, store the checksum of the file as well.
+     suffix: The suffix of the handle. Default: ''.
+     Returns the URI of the new handle, raises a 503 exception if an error occurred.
```

Comment:
Could you please fix the comments here so that they follow the google comment styling (which we use with the napoleon sphinx plugin) http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments
https://pypi.python.org/pypi/sphinxcontrib-napoleon

```
     Returns:
        the URI of the new handle, raises a 503 exception if an error occurred.
```